frontend/src/styles/ProductDetailStyles.tsx

```diff
@@ -0,0 +1,11 @@
+import { makeStyles } from "@material-ui/core";
+
+export const useProductDetailStyles = makeStyles({
```

Comment:
You can use a different naming convention for the styling files. That would be using the `.styles` extension instead of adding Styles into the name. So it would look like this: `ProductDetails.styles.tsx`. This naming convention is similar to the Angular one